tests/host_tools/cargo_build.py

```diff
@@ -57,43 +60,27 @@ def cargo_test(path, extra_args='', extra_env=''):
     run(cmd, shell=True, check=True)
 
 
-def get_firecracker_binaries(root_path, features=''):
+def get_firecracker_binaries(features=''):
     """Build the Firecracker and Jailer binaries if they don't exist.
 
     Returns the location of the firecracker related binaries eventually after
     building them in case they do not exist at the specified root_path.
     """
-    extra_args = \
-        '--release --target {} >/dev/null 2>&1'.format(DEFAULT_BUILD_TARGET)
-    extra_env = 'TARGET_CC=musl-gcc'
-
-    if features == '':
-        cargo_binaries_rel_path = CARGO_RELEASE_REL_PATH
-    else:
-        raise UnknownFeatureException
-
-    path_to_binaries = os.path.join(
-        root_path,
-        os.path.join(
-            cargo_binaries_rel_path,
-            RELEASE_BINARIES_REL_PATH
-        )
+    if features != '':
```

Comment:
We should probably just get rid of features. If we add features in the future we can re-add this logic. What do you think?

Reply:
I agree. Removed.